e2e-tests/cypress/tests/integration/channels/profile_settings/profile_settings_spec.js

```diff
@@ -48,4 +48,61 @@ describe('Profile Settings', () => {
         // # Close the modal
         cy.uiClose();
     });
+
+    const fileTypes = [
+        {
+            extension: 'PNG',
+            fileName: 'profile_picture.png',
+        },
+        {
+            extension: 'JPG',
+            fileName: 'profile_picture.jpg',
+        },
+        {
+            extension: 'JPEG',
+            fileName: 'profile_picture.jpeg',
+        },
+        {
+            extension: 'BMP',
+            fileName: 'profile_picture.bmp',
+        },
+    ];
+
+    fileTypes.forEach((fileType) => {
+        it(`MM-T2078 Profile picture: file ${fileType.extension} type accepted`, () => {
```

Comment:
Please include the index so that each test (`it`) block can represent per step of the test case and can be saved properly in our test management system: will become:
```suggestion
    fileTypes.forEach((fileType, index) => {
        it(`MM-T2078_${index + 1} Profile picture: file ${fileType.extension} type accepted`, () => {
```